src/Actions/sendPasswordReset.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Lab19\Cart\Actions;
+
+use Illuminate\Support\Carbon;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Password;
+use Lab19\Cart\Models\PasswordResets;
+use Lab19\Cart\Models\User;
+use Lab19\Cart\Notifications\GernzyResetPassword;
+
+class SendPasswordReset
```

Comment:
@AlexeiaW  We just need to match the filename here with the case and then this PR / tests should pass.